Application/Interfaces/IAuthService.cs

```diff
@@ -0,0 +1,12 @@
+﻿using Application.DTOs;
+
+namespace Application.Interfaces;
+
+public interface IAuthService
+{
+    Task<LoginUserReturnDto> LoginUserAsync(LoginUserDto userLoginDto);
+
+    Task<bool> ValidateEmailAsync(string email);
+
+    Task<LoginUserReturnDto> RegisterUserAsync(RegisterUserDto userRegisterDto);
```

Comment:
```suggestion
    Task<LoginUserReturnDto> LoginUserAsync(LoginUserDto loginUserDto);

    Task<bool> ValidateEmailAsync(string email);

    Task<LoginUserReturnDto> RegisterUserAsync(RegisterUserDto registerUserDto);
```
(opisane w innym komentarzu, czemu tak)